### PR TITLE
#11 Adds links to forms on landing page

### PR DESCRIPTION
**Pull request by Cleop**
#11 Adds links to forms on landing page for newsletter and join club soda

To be merged after #149

### CODE REVIEW

File: lib/cs_guide/resources/drink.ex [hunk 1]
```diff
@@ -11,7 +11,7 @@ defmodule CsGuide.Resources.Drink do
     field(:abv, :float)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
-    field(:description, :string)
+    field(:description, :string, default: "")
```
Comment (nelsonic) [inline lib/cs_guide/resources/drink.ex]: @Danwhy are you still planning on adding the `Fields.description` with XSS protection?

File: lib/cs_guide/resources/venue.ex [hunk 2]
```diff
@@ -12,8 +12,16 @@ defmodule CsGuide.Resources.Venue do
     field(:postcode, :string)
     field(:phone_number, :string)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
+    field(:description, :string)
+    field(:website, :string)
+    field(:address_line_1, :string)
```
Comment (nelsonic) [inline lib/cs_guide/resources/venue.ex]: Please update to `field(:address, Fields.address)` once package published.

File: lib/cs_guide_web/templates/page/index.html.eex [hunk 3]
```diff
@@ -58,13 +58,13 @@
         Know the best pub for low and no alcohol beers? Found a great bar with
         alcohol free cocktails? Let us know and we will get them listed on the Guide.
       </p>
-      <a class="absolute-horizontal-center-s absolute-vertical-center-ns bottom-4-s right-2-ns bg-white br2 ph4 pv2 mr5-ns cs-black no-underline lh4 f4 w5" href="#">Nominate a Venue</a>
+      <a class="absolute-horizontal-center-s absolute-vertical-center-ns bottom-4-s right-2-ns bg-white br2 ph4 pv2 mr5-ns cs-black no-underline lh4 f4 w5" href="https://clubsoda.typeform.com/to/Z5sMPA">Nominate a Venue</a>
```
Comment (nelsonic) [inline lib/cs_guide_web/templates/page/index.html.eex]: mixing typeform with a Phoenix app is admitting "defeat" from a dev/UX perspective. 
it pains me to merge this. but if it's what the PO wants, ship!

**Comment by Cleop** [comment]
Thanks for the feedback re the branch name @nelsonic, noted.

**Pull request merged**